src/haddock/modules/topology/topocg/aa2cg.py

```diff
@@ -399,7 +402,27 @@ def extract_groups(pair_list):
     out.close()
 
 
-def determine_ss(structure, skipss, pdbf_path):
+def create_file_with_cryst(pdb_file: str) -> None:
+    """
+    This function creates a new pdb because the CRYST line is missing from the pdf file.     
+    This line is necessary for DSSP.
+
+    Args:
+        output: str
+        pdb_file: str
+
+    Returns:
+        pdb_file_copy: str
+
+    """
+    with open(pdb_file, "r") as file_in, tempfile.NamedTemporaryFile(mode = "w", delete=False) as file_out:
+        content = file_in.read()
+        file_out.write(CRYST_LINE + content)
+    
+    return(file_out.name)
+
+
+def determine_ss(structure, output_path, skipss, pdbf_path):
```

Comment:
I think `output_path` is not used in this function anymore

Reply:
yes I forgot to remove it !